ModernWallet/Classes/ViewController/DimPresentation/DimPresentationAnimator.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  DimPresentationAnimator.swift
+//  SOCOTAB
+//
+//  Created by Georgi Ivanov on 22.02.18.
+//  Copyright © 2018 Primeholding AD. All rights reserved.
+//
+
+import UIKit
+
+class DimPresentationAnimator: NSObject {
+    let isPresenting: Bool
+    var duration: TimeInterval = 0.7
```

Comment:
This variable is used only within the scope of the class. Move it to the extension as a private computed property

```extension DimPresentationAnimator: UIViewControllerAnimatedTransitioning {
    fileprivate var duration: TimeInterval {
        return 0.7
    }
// the rest of the extension
```